src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -506,7 +527,7 @@ const DiscoveryActionBar = (props: Props) => {
             }
             type='default'
             className={`discovery-action-bar-button${(props.discovery.selectedResources.length === 0) ? '--disabled' : ''}`}
-            disabled={props.discovery.selectedResources.length === 0 || downloadStatus.inProgress}
+            disabled={props.discovery.selectedResources.length === 0 || downloadStatus.inProgress || healICPSRLoginNeeded}
```

Comment:
I think we should move this line of logic outside because it will be more complicated than this one-liner
The expected behavior should be: 
1. If user is not logged in, this button is enabled, the HEAL-ICPSR tooltip can be visible, clicking on this button redirects to the login page
2. If user is logged in via non-incommons IDP, this button is disabled, the HEAL-ICPSR tooltip is visible
3. If user is logged in via incommons IDP, this button is enabled, the HEAL-ICPSR tooltip is invisible

Reply:
should be fixed now